src/Controller/BlogController.php

```diff
@@ -151,29 +144,45 @@ public function showASinglePost(int $postId)
 
         self::render(self::VIEW_BLOG_POST, [
             'post' => $post,
-            'categories' => $categories
+            'categories' => $categories,
+            'connectedMember' => isset($_SESSION['connected-member']) ? $_SESSION['connected-member'] : null
         ]);
     }
 
     /**
      * Show the panel do manage blog posts
      *
      * @param string $message
+     * @param array $yesNoForm
+     * @throws BlogException
      * @throws \Twig_Error_Loader
      * @throws \Twig_Error_Runtime
      * @throws \Twig_Error_Syntax
+     * @throws AccessException
      */
-    public function showAdminPanel(string $message = '')
+    public function showAdminPanel(string $message = '', array $yesNoForm = [])
     {
         $posts = $this->postManager->getAll();
         $tags = $this->tagManager->getAll();
         $categories = $this->categoryManager->getAll();
 
+        if (isset($_SESSION['connected-member'])) {
```

Comment:
it feels like you will use this function a lot, not only here, it could be great if you can find a better place (AccessManagement::denyAccessUnlessGranted('admin'))
or something like this